libhb/scan.c

```diff
@@ -1376,6 +1376,12 @@ static void LookForAudio(hb_scan_t *scan, hb_title_t * title, hb_buffer_t * b)
         }
     }
 
+	// Append input bitrate in kbps to the end of the description
+	char in_bitrate_str[12];
+	sprintf(in_bitrate_str, " %d", audio->config.in.bitrate / 1000);
```

Comment:
Also, use snprintf to avoid bad input causing an overrun of in_bitrate_str

Reply:
Is that necessary? The config.in.bitrate integer can't be longer than 10 characters by design, including the space and string terminator 12 should be enough.